servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java

```diff
@@ -137,28 +158,7 @@ public abstract HttpServerBuilder enableWireLogging(String loggerName, LogLevel
      * @param transportObserver A {@link TransportObserver} that provides visibility into transport events.
      * @return {@code this}.
      */
-    public abstract HttpServerBuilder transportObserver(TransportObserver transportObserver);
-
-    /**
-     * Disables automatic consumption of request {@link StreamingHttpRequest#payloadBody() payload body} when it is not
-     * consumed by the service.
-     * <p>
-     * For <a href="https://tools.ietf.org/html/rfc7230#section-6.3">persistent HTTP connections</a> it is required to
-     * eventually consume the entire request payload to enable reading of the next request. This is required because
-     * requests are pipelined for HTTP/1.1, so if the previous request is not completely read, next request can not be
-     * read from the socket. For cases when there is a possibility that user may forget to consume request payload,
-     * ServiceTalk automatically consumes request payload body. This automatic consumption behavior may create some
-     * overhead and can be disabled using this method when it is guaranteed that all request paths consumes all request
-     * payloads eventually. An example of guaranteed consumption are {@link HttpRequest non-streaming APIs}.
-     *
-     * @return {@code this}.
-     * @deprecated Use {@link #drainRequestPayloadBody(boolean)}.
-     */
-    @Deprecated
-    public final HttpServerBuilder disableDrainingRequestPayloadBody() {
```

Comment:
Please, don't mix this PR with unrelated changes. Consider submitting a separate PR to clean up deprecations. Similar to what Scott did: https://github.com/apple/servicetalk/pull/1805
It will help to highlight all meaningful changes as separate items on release notes, and keep the scope of each PR independent.

Reply:
Brought back the method. Removal can follow after this is merged.

Reply:
I see that you have the same unrelated change in client builder PR: https://github.com/apple/servicetalk/pull/1819/files#diff-f220503d52fc93c626fb3cbfc3a3814f449420c90bc980ae509d40af7e7de085L100
Consider cleaning up all deprecated methods (for client/server http/grpc) builders before these conversions. Going forward, we should clean up deprecated methods as soon as those are cherry-picked into 0.41.

Reply:
Agreed, please approve: https://github.com/apple/servicetalk/pull/1844